pgwatch2/metrics/wal/10/metric.sql

```diff
@@ -8,5 +8,11 @@ select /* pgwatch2_generated */
     end as xlog_location_b,
   case when pg_is_in_recovery() then 1 else 0 end as in_recovery_int,
   extract(epoch from (now() - pg_postmaster_start_time()))::int8 as postmaster_uptime_s,
-  system_identifier::text as tag_sys_id
+  system_identifier::text as tag_sys_id,
+  case
+    when pg_is_in_recovery() = false then
```

Comment:
is there any idea behind this statement? Why not just `NOT pg_is_in_recovery()`?

Reply:
no big reason besides my own understanding of what is more readable SQL :D "false" jumps out more to me...although yeah technically unnecessary and could be changed...not sure if worth it tho :)

Reply:
no problem. Just thought maybe some tuning trick :)